client/components/UserProfile.jsx

```diff
@@ -69,17 +54,19 @@ function UserProfile(){
         {clubs.map((club) => (
           // key needs to be unique id
           <li key={club.club_id}>
-            <span>name: {club.name}</span>{' '}
+            <span>name: {club.clubName}</span>{' '}
             <span>description: {club.description}</span>
-            <button onClick={() => getBookClub(club.club_id)}>Open Book Club</button>
+            {/* <button onClick={() => getBookClub(club.club_id)}>Open Book Club</button> */}
+            <button onClick={() => navigate(`/${club.club_id}`)}>Open Book Club</button>
           </li>
         ))}
       </ul>
       <button onClick = {() => setShow(!show)}>Create New Club</button>
-      { show && <form style={{display: 'flex', flexDirection: 'column'}} id='createclub' onSubmit={(e) => createClub(e)} >
+      { show && <form id='createclub' onSubmit={(e) => createClub(e)} >
```

Comment:
Adding to the create club function that on success it calls setShow(!show) it should hide the form.